### DISCUSSION TITLE
Mountain Duck: How To Know What Is Left To Sync?

### DISCUSSION THREAD

**Discussion by WhatImListnTo** [iterate-ch/cyberduck#16138] [Q&A]
I am searching all over to find out how can I possibly know what is left to sync for Mountain Duck offline storage.  What I did was left click Windows Explorer root for the Mountain Duck drive, click Mountain Duck -> Info, General tab followed by clicking Calculate button.  This ran for a very long time to finally show me:

![Mountain_Duck_JJtrui8O22__13_20_35_00](https://github.com/user-attachments/assets/abf44ffa-9813-4a98-b9f1-62d8355dcf63)

This shows 1.2 TiB total size that matches what I have uploaded to my SFTP server.  However, Mountain Duck is constantly showing me pending files to be downloaded in Sync in Progress context menu of the task bar icon.  The remaining time appears to be for a specific directory only.

Is there anyway I can know how much is left in total to be sync'd?

**Comment by tatze96** [comment]
We can't provide a time estimate for the whole synchronization queue, which is why there is only a time estimate displayed for items within the current synchronization batch. The number and total size of the remaining items within the queue are displayed below the "Sync in Progress" indication, as shown within the screenshot on our [wiki page](https://docs.cyberduck.io/mountainduck/connect/sync/#sync-progress).
